operations/validate_csm_health.md

```diff
@@ -116,21 +116,7 @@ See [Known issues with NCN resource checks](../troubleshooting/known_issues/ncn_
 
 ### 1.3 Check of system management monitoring tools
 
-If all designated prerequisites are met, the availability of system management health services may optionally be validated by accessing the URLs listed in
-[Access System Management Health Services](system_management_health/Access_System_Management_Health_Services.md).
```

Comment:
The automated test suite is not doing the checks that are listed in the "Access SMHS" link here. Do we want to remove the link to these optional checks even though the automated script we're adding does not cover them?